build/tasks.js

```diff
@@ -30,45 +30,45 @@ export default {
   },
 
   async run(args = []) {
-    await this.build();
+    await this.build([...args, '--force']);
```

Comment:
This is fixing #357 and I think it's fine to do it like this for now. What might be nice in the future is to load build-config first, then diff it against the new build-config and in that case turn force on.